packages/KStepper/stepper-icons/KPendingState.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="k-step-spinner">
+    <svg
+      :width="size"
+      :height="size"
+      viewBox="0 0 50 50"
+    >
+      {{ title }}
+      <circle
+        cx="25"
+        cy="25"
+        r="20"
+        fill="none"
+        stroke-width="5"
+      />
+    </svg>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'KPendingState',
+  props: {
+    size: {
+      type: String,
+      default: '24'
+    },
+    title: {
+      type: String,
+      default: 'Completed'
```

Comment:
I think this is the wrong default title for this icon

Reply:
Fixed